e2e/test/scenarios/embedding/reproductions/37914-41635-embed-preview-parameter-values.cy.spec.js

```diff
@@ -0,0 +1,233 @@
+    );
+
+    restore();
+    cy.signInAsAdmin();
+  });
+
+  it("dashboard linked filters values don't work in static embed preview (metabase#37914)", () => {
+    const dashboardDetails = {
+      parameters: [filter, filter2, filter3],
+      enable_embedding: true,
+      embedding_params: {
+        [filter.slug]: "enabled",
+        [filter2.slug]: "enabled",
+        [filter3.slug]: "enabled",
+      },
+    };
+    cy.createQuestionAndDashboard({
+      questionDetails,
+      dashboardDetails,
+    }).then(({ body: { card_id, dashboard_id } }) => {
+      addOrUpdateDashboardCard({
+        dashboard_id,
+        card_id,
+        card: {
+          parameter_mappings: [
+            {
+              card_id,
+              parameter_id: filter.id,
+              target: [
+                "dimension",
+                ["field", PRODUCTS.CATEGORY, { "base-type": "type/Text" }],
+              ],
+            },
+            {
+              card_id,
+              parameter_id: filter2.id,
+              target: [
+                "dimension",
+                ["field", PRODUCTS.CATEGORY, { "base-type": "type/Text" }],
+              ],
+            },
+            {
+              card_id,
+              parameter_id: filter3.id,
+              target: [
+                "dimension",
+                ["field", PRODUCTS.CATEGORY, { "base-type": "type/Text" }],
+              ],
+            },
+          ],
+        },
+      });
+
+      visitDashboard(dashboard_id);
+    });
+
+    openStaticEmbeddingModal({
+      activeTab: "parameters",
+      previewMode: "preview",
+    });
+
+    modal().within(() => {
+      // Makes it less likely to flake.
+      cy.wait("@previewDashboard");
+
+      getIframeBody().within(() => {
+        cy.log(
+          "Set filter 2 value, so filter 1 should be filtered by filter 2",
+        );
+        cy.button(filter2.name).click();
+        cy.wait("@previewValues");
+        popover().within(() => {
+          cy.findByText("Gadget").should("be.visible");
+          cy.findByText("Gizmo").should("be.visible");
+          cy.findByText("Widget").should("be.visible");
+          cy.findByText("Doohickey").click();
+          cy.button("Add filter").click();
+        });
+
+        cy.log("Assert filter 1");
+        cy.button(filter.name).click();
+        popover().within(() => {
+          cy.findByText("Gadget").should("not.exist");
+          cy.findByText("Gizmo").should("not.exist");
+          cy.findByText("Widget").should("not.exist");
+          cy.findByText("Doohickey").should("be.visible");
+        });
+      });
+    });
+  });
+
+  it("dashboard linked filters values in embed preview don't behave like embedding (metabase#41635)", () => {
+    const dashboardDetails = {
+      parameters: [filter, filter2, filter3],
+      enable_embedding: true,
+      embedding_params: {
+        [filter.slug]: "enabled",
+        [filter2.slug]: "locked",
+        [filter3.slug]: "locked",
```

Comment:
Set both to locked as described in https://github.com/metabase/metabase/issues/41635